app/controllers/people/index.js

```diff
@@ -34,14 +36,26 @@ export default class PeopleIndexController extends Controller {
 
   get hasNoResults() {
     return (
-      this.model.loadPeopleTaskInstance.isFinished && this.people.length === 0
+      this.model?.loadPeopleTaskInstance?.isFinished &&
```

Comment:
Why is the optional chaining needed here? Normally the controller always has the model set before the page is rendered.